### PR TITLE
Add operate first data science community subproject

### PR DESCRIPTION
**Pull request by aakankshaduggal**
This pull request adds Operate First Data Science Community subproject to the Data Science SIG.

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -22,6 +22,12 @@ sigs:
     tz: ET (East Time)
     frequency: bi-weekly
     url: meet.google.com/ufs-hgvi-oni
+  - description: Data Science SIG - Operate First Data Science Community Meeting
```
Comment (MichaelClifford) [inline sigs.yaml:25]:
```suggestion
  - description: Operate First Data Science Community Project Meeting
```

File: sigs.yaml [hunk 2]
```diff
@@ -22,6 +22,12 @@ sigs:
...
+    day: Wednesday
+    time: "10:00"
+    tz: ET (East Time)
+    frequency: bi-weekly
```
Comment (MichaelClifford) [inline sigs.yaml:29]:
```suggestion
    frequency: weekly
```

File: sig-data-science/README.md [hunk 3]
```diff
@@ -34,8 +34,14 @@ The Chairs of the SIG run operations and processes governing the SIG.
 
 The following [subprojects][subproject-definition] are owned by sig-data-science:
 ### AIOps
-- **Owners:**
+**Owners:**
   - [open-services-group/community/sig-data-science](https://github.com/open-services-group/community/blob/main/sig-data-science/OWNERS)
+
+  #### 1. Operate First Data Science Community
+   - The Operate First Data Science Community aims to embrace the multidisciplinary nature of the current state of AI operations and cloud infrastructure by discussing technical topics relevant to data scientists, software developers, DevOps professionals, statisticians, and the intersection of their work, all in an open and collaborative public platform.
+
```
Comment (MichaelClifford) [inline sig-data-science/README.md:42]: This document is auto-generated so placing content here might mess it up.  You will need to add a section below and move these subproject details there. please see this example:  https://github.com/kubernetes/community/blob/master/sig-architecture/README.md

**Review by MichaelClifford — CHANGES_REQUESTED** [review]

File: sig-data-science/README.md (hunk 3 above)
Comment (durandom) [reply sig-data-science/README.md:42]: You add content to https://github.com/open-services-group/community/blob/main/sigs.yaml#L30
and then this template is used
https://github.com/open-services-group/community/blob/main/generator/sig_readme.tmpl

see the template for available variables

Comment (MichaelClifford) [reply sig-data-science/README.md:42]: I dont think the the subproject description content is auto generated by sig_readme.tmpl. See the `<!-- BEGIN CUSTOM CONTENT -->` line below the generated content here, https://raw.githubusercontent.com/kubernetes/community/master/sig-architecture/README.md

Comment (MichaelClifford) [reply sig-data-science/README.md:42]: Here is some more docs on how to add custom content to the README: 
https://github.com/kubernetes/community/blob/master/generator/README.md#adding-custom-content 

@aakankshaduggal can you please move this content to the "custom content" section of the readme?

Comment (aakankshaduggal) [reply sig-data-science/README.md]: Done :+1:

File: sigs.yaml [hunk 4]
```diff
@@ -30,6 +36,9 @@ sigs:
   - name: AIOps
     owners:
     - https://raw.githubusercontent.com/open-services-group/community/main/sig-data-science/OWNERS
+  - name: Operate First Data Science Community
+    owners:
```
Comment (MichaelClifford) [inline sigs.yaml:40]:
```suggestion
     '[Described below](#operate-first-data-science-community)'
    owners:
```

File: sig-data-science/README.md [hunk 5]
```diff
@@ -46,4 +47,12 @@ The following [subprojects][subproject-definition] are owned by sig-data-science
 [subproject-definition]: https://github.com/open-services-group/community/blob/master/governance.md#subprojects
 <!-- BEGIN CUSTOM CONTENT -->
 
+# Details about SIG Data Science sub-projects
+
+## Operate First Data Science Community
+
+The Operate First Data Science Community aims to embrace the multidisciplinary nature of the current state of AI operations and cloud infrastructure by discussing technical topics relevant to data scientists, software developers, DevOps professionals, statisticians, and the intersection of their work, all in an open and collaborative public platform.
```
Comment (MichaelClifford) [inline sig-data-science/README.md:54]:
```suggestion
This subproject is responsible for organizing, hosting, promoting and growing the Operate First Data Science Community and its associated meetup.  

The  Operate First Data Science Community aims to embrace the multidisciplinary nature of the current state of AI operations and cloud infrastructure by discussing technical topics relevant to data scientists, software developers, DevOps professionals, statisticians, and the intersection of their work, all in an open and collaborative public platform.
```

**Comment by MichaelClifford** [comment]
:+1: Thanks @aakankshaduggal 
LGTM

**Review by schwesig — APPROVED** [review]
/lgtm

**Pull request merged**